working-groups/comms/charter.md

```diff
@@ -0,0 +1,61 @@
+
+## Goals
+- Establish the marketing strategy for TAG ENV, and then execute that strategy in areas such as Events, Digital Media, Public Relations, Web Content, and Branding. 
+- Collaborate with other sustainability projects to produce sustainability partnerships and marketing materials.
+- Communicate the results of the projects’ cloud native sustainability reviews.
+
+## Scope
+- Act within the CNCF Environmental Sustainability TAG [Charter](https://github.com/cncf/tag-env-sustainability/blob/main/charter.md). 
+- Create TAG awareness marketing strategy
+- Establish thought leadership within the TAG
+- Monitor Sustainability events
+- Collaborate with other open source sustainability projects
+- Outreach and marketing are scoped TAG wide and include WGs
+
+## Not in Scope
+- Marketing sustainability products or solutions from vendors
+- Marketing for other community groups outside the TAG ENV
+
+## Deliverables
+- Create templates for contributions to blogs and white papers
+- Maintain event calendar
+- Website maintenance/refresh
+- Social Media Calendar
+- Social Media Posts
+- Content (blogs, white papers, website copy, etc.)
+- Plan in-person events at selected conferences
+
+## Governance
+The Contributor Comms subproject welcomes and encourages its members to take on leadership roles in their contributions. The group defines the areas of its work such as social media, blogging, etc. and creates associated leadership roles accordingly. Members are encouraged to pursue any interest in specific areas as contributing team members, leads, or shadows. Leadership roles are defined in the Kubernetes [subproject’s role handbooks](https://github.com/kubernetes/community/tree/master/communication/contributor-comms/role-handbooks).
+- Recommend two co-chairs
+- Recommend sub chairs for events, content, website and social media
+
+## Roles and Subproject Management
+General subproject leadership is handled by the Contributor Comms Subproject Lead(s). Specific subproject responsibilities are also broken down into areas with their own lead roles, as defined here:
+- [Contributor Comms Subproject Lead](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/Subproject-Lead.md)
+- [Social Media Coordinator](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/Social-Media.md)
+- [Comms Tech Lead](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/Comms-Tech-Lead.md)
+- [Comms Blog Coordinator](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/role-handbooks/blog-coordinator.md)
+- Emeritus (previous team members that we had the privilege of working with 
+
+All roles can have multiple shadows and should have at least one shadow. All leads should actively recruit members and shadows for the entire team.
+
+Roles are flexible and subject to change.
+
```

Comment:
```suggestion
```

Reply:
In the comment above I recommended listing subteams with a one sentence description. Further operation details of the team has to be defined in handbooks. (`workinggroups/comms/handbooks/***`)